ReportHPXMLOutput/tests/hpxml_output_report_test.rb

```diff
@@ -112,7 +139,34 @@ def test_air_source_heat_pump_xml
       'Systems: Mechanical Ventilation Flow Rate (cfm)' => 0.0,
       'Primary Systems: Heating Capacity (kBtu/h)' => 36.0,
       'Primary Systems: Cooling Capacity (kBtu/h)' => 36.0,
-      'Primary Systems: Heat Pump Backup Capacity (kBtu/h)' => 36.0
+      'Primary Systems: Heat Pump Backup Capacity (kBtu/h)' => 36.0,
+      'Design Loads Heating: Total (Btu/h)' => 31214.0,
```

Comment:
It would be nice to be consistent between HVAC design loads and HVAC capacities. I suggest we stick with the units from the HPXML schema -- can you update the HVAC capacities to be Btu/h instead of kBtu/h?